CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 
 ### Bug Fixes
 
+* [#2123](https://github.com/bbatsov/rubocop/pull/2123): Fix handing of dynamic widths `Lint/FormatParameterMismatch`. ([@edmz][])
```

Comment:
Well, not all of them were filed as bugs, like the missing 'A' formatter. Or the heredoc.